Model/Api/OrderManagement.php

```diff
@@ -166,6 +167,14 @@ public function manage(
                 $this->saveUpdateOrder($reference, $type, $display_id, $storeId);
             }
             $this->metricsClient->processMetric("webhooks.success", 1, "webhooks.latency", $startTime);
+        } catch (BoltException $e) {
+            $this->bugsnag->notifyException($e);
+            $this->metricsClient->processMetric("webhooks.failure", 1, "webhooks.latency", $startTime);
+            $this->response->setHttpResponseCode(422);
+            $this->response->setBody(json_encode([
+                'status' => 'failure',
+                'error' => ['code' => $e->getCode(), 'message' => $e->getMessage()],
```

Comment:
I've changed the error answer format.
With "old" format we have in DD:
![image](https://user-images.githubusercontent.com/7270718/74734208-c096e100-5256-11ea-9861-fc42cd55f62e.png)

With new one:

![image](https://user-images.githubusercontent.com/7270718/74734243-d3111a80-5256-11ea-9e1e-1927f95344a1.png)